nengo/networks/workingmemory.py

```diff
@@ -5,15 +5,13 @@
 
 
 def InputGatedMemory(n_neurons, dimensions, fdbk_scale=1.0, gate_gain=10,
-                     difference_gain=1.0, reset_gain=3,
-                     mem_config=None, net=None):
+                     difference_gain=1.0, reset_gain=3, net=None):
     """Stores a given vector in memory, with input controlled by a gate."""
     if net is None:
         net = nengo.Network(label="Input Gated Memory")
 
-    if mem_config is None:
-        mem_config = nengo.Config(nengo.Connection)
-        mem_config[nengo.Connection].synapse = nengo.Lowpass(0.1)
+    mem_config = nengo.Config(nengo.Connection)
+    mem_config[nengo.Connection].synapse = nengo.Lowpass(0.1)
```

Comment:
:+1:

On 20 August 2015 at 12:48, tcstewar <notifications@github.com> wrote:

> In nengo/networks/workingmemory.py
> <https://github.com/nengo/nengo/pull/814#discussion_r37552615>:
>
> >      """Stores a given vector in memory, with input controlled by a gate."""
> >      if net is None:
> >          net = nengo.Network(label="Input Gated Memory")
> >
> > -    if mem_config is None:
> > -        mem_config = nengo.Config(nengo.Connection)
> > -        mem_config[nengo.Connection].synapse = nengo.Lowpass(0.1)
> > +    mem_config = nengo.Config(nengo.Connection)
> > +    mem_config[nengo.Connection].synapse = nengo.Lowpass(0.1)
>
> Cool... I'm fine with @xchoo <https://github.com/xchoo> outvoting use 1
> to 2.... especially since we're getting fewer parameters anyway, so having
> one weird one isn't too bad. :)
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/nengo/nengo/pull/814/files#r37552615>.
>